shared-helpers/src/programHelpers.ts

```diff
@@ -0,0 +1,58 @@
+import { ApplicationProgram, Program } from "@bloom-housing/backend-core/types"
+
+export const PROGRAMS_FORM_PATH = "application.programs"
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const mapProgramsToApi = (programs: Program[], data: Record<string, any>) => {
```

Comment:
In testing this with the seeded data, it appears that this function in practice always takes data with one key like `{tay: 'preferNotToSay'}` and returns an array with one element like `[{key: 'tay', claimed: true, options: Array(3)}]`.  This may be operating off an old model of having multiple programs/preferences per page, so that would make sense. Given that there is only one coming in and out of this function in reality, we can probably simplify this and perhaps code further upstream and downstream as well.

Reply:
Good point, I can simplify that.